### ISSUE TITLE
neon installation broken inside of docker

### ISSUE THREAD

**Issue by deepio** [DDMAL/neon2-wrapper#13]
# Neon2-wrapper (develop branch)

None of the build steps have changed 
```shell
# Install Neon
git clone --recurse-submodules -b "$BRANCHES" https://github.com/DDMAL/neon2-wrapper
cd ./neon2-wrapper
git submodule update --init
yarn install
yarn build
cd /code/jobs
```

But now we can not build because of Neon
![Screen Shot 2019-05-08 at 11 15 08 AM](https://user-images.githubusercontent.com/28245482/57386767-06306480-7183-11e9-826f-436abe3bd07a.png)
The static folder does not exists beyond the neon2-wrapper folder.

**Comment by JRegimbal** [comment]
Should be fixed in 16eb202.

**Comment by deepio** [comment]
Thank you.
